extensions/guacamole-auth-cas/src/main/resources/translations/en.json

```diff
@@ -0,0 +1,7 @@
+{
+
+    "LOGIN" : {
+        "FIELD_HEADER_TICKET" : ""
```

Comment:
Hm ... This issue will likely bite me, as well, with the OpenID dev. In fact, I suspect it may have already and I just don't remember. It's been a bit since I actively worked with that code.

I wonder if perhaps we should modify the handling of fields such that fields without defined templates do not render at all... That way automatically-populated fields required for credentials, etc. could be defined without requiring repeated CSS or empty translation string hackery.